crates/rpc/src/lib.rs

```diff
@@ -189,6 +194,8 @@ impl RpcServer {
             .with_state(v07_routes.clone())
             .route("/rpc/v0_8", post(rpc_handler).get(rpc_handler))
             .with_state(v08_routes.clone())
+            .route("/rpc/v0_9", post(rpc_handler).get(rpc_handler))
+            .with_state(v09_routes.clone())
             .route("/rpc/pathfinder/v0.1", post(rpc_handler))
             .route("/rpc/pathfinder/v0_1", post(rpc_handler))
```

Comment:
I didn't change this, but coming to think about it, shouldn't the route be declared just once?

Reply:
> Which route do you mean?

Oh, "/rpc/pathfinder/v0.1" is _not_ the same as "/rpc/pathfinder/v0_1"... :-)